main.py

```diff
@@ -2,6 +2,30 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+    for genre in genres:
+        Genre.objects.create(name=genre)
+
+    actors = [("George", "Klooney"),
+              ("Kianu", "Reaves"),
```

Comment:
The name 'Kianu Reaves' seems to be a typo. It is later corrected to 'Keanu Reeves', but it would be more efficient to use the correct spelling initially.